akka-cluster-sharding/src/main/scala/akka/cluster/sharding/internal/EventSourcedRememberEntities.scala

```diff
@@ -27,12 +27,17 @@ import akka.persistence.SnapshotSelectionCriteria
  * INTERNAL API
  */
 @InternalApi
-private[akka] final class EventSourcedRememberEntitiesStoreProvider(typeName: String, settings: ClusterShardingSettings)
-    extends RememberEntitiesShardStoreProvider {
+private[akka] final class EventSourcedRememberEntitiesProvider(typeName: String, settings: ClusterShardingSettings)
+    extends RememberEntitiesProvider {
 
+  // this is backed by an actor using the same events as for state-store-mode=persistence
   override def shardStoreProps(shardId: ShardId): Props =
     EventSourcedRememberEntitiesStore.props(typeName, shardId, settings)
 
+  // FIXME persistent state store deprecated but we are adding a remember entities store that is not deprecated
+  // We need a new impl for this to allow ddata + persistent remember entities
+  // For now it is anyways not possible to configure state store and remember entities store separately so this is never used
+  override def coordinatorStoreProps(): Props = ???
```

Comment:
what does this mean compared to master? what happens now when mode=persistence && rememberEntities=true ?

Reply:
`mode=persistence` means the persistent actor subclass of coordinator is used, that impl is left as is and uses its own state rather than an external store for remember entities, so this is never called.